src/ctia/lib/riemann.clj

```diff
@@ -0,0 +1,124 @@
+(ns ctia.lib.riemann
+  (:require [clojure.string :as str]
+            [clojure.tools.logging :as log]
+            [riemann.client :as riemann]
+            [ctia.properties :as prop]
+            [ctia.lib.utils :as utils])
+  (:import [clojure.lang ExceptionInfo]))
+
+(defn request->event
+  [request extra-fields]
+  (into {:uri (str (:uri request))
+         :_params (utils/safe-pprint (:params request))
+         :remote-addr (str (if-let [xff (get-in request [:headers "x-forwarded-for"])]
+                             (peek (str/split xff #"\s*,\s*"))
+                             (:remote-addr request)))
+         :request-headers (prn-str (:headers request))
+         :request-body (let [;; HttpInputOverHTTP => string
+                             bstr (pr-str (:body request))]
+                         (subs bstr 0 (min (count bstr) 100)))
+         :request-method (str (:request-method request))
+         :mask (str (= "Mask" (get-in request [:headers "x-client-app"])))
```

Comment:
`Mask` is the Threat Grid frontend. It has no meaning in our context

Reply:
Deleted.